charts/datadog/README.md

```diff
@@ -544,6 +544,14 @@ helm install <RELEASE_NAME> \
 | agents.volumeMounts | list | `[]` | Specify additional volumes to mount in all containers of the agent pod |
 | agents.volumes | list | `[]` | Specify additional volumes to mount in the dd-agent container |
 | clusterAgent.additionalLabels | object | `{}` | Adds labels to the Cluster Agent deployment and pods |
+| clusterAgent.admissionController.agentSidecarInjection.clusterAgentEnabled | bool | `true` | Enable communication between Agent sidecars and Cluster Agent. |
```

Comment:
i may have missed it, i don't see this parameter referenced in the templates. it also seems redundant with `.enabled` a couple lines below

Reply:
Good catch, I missed this in the template! 
This is not same as `clusterAgent.admissionController.agentSidecarInjection.enabled` which enabled the sidecar injection. `clusterAgentEnabled` enables communication between sidecar agents and DCA.

Reply:
i see. sorry i read that a few times and didn't register until now.

i wonder if we could name it so it's a little more intuitive; `enable` is associated with `clusterAgent` elsewhere with a different meaning. how about `clusterAgentLinked`, or `clusterAgentCommunicationEnabled`, or `clusterAgentListen`?

Reply:
`clusterAgentCommunicationEnabled` sounds good, will update.